From: https://sources.debian.net/src/lvm2/2.02.111-2.2/debian/patches/install.patch/
Index: LVM2.2.02.120/make.tmpl.in
===================================================================
--- LVM2.2.02.120.orig/make.tmpl.in
+++ LVM2.2.02.120/make.tmpl.in
@@ -441,17 +441,14 @@ install_lib_shared: $(LIB_SHARED)
 	$(INSTALL_DIR) $(usrlibdir)
 	$(LN_S) -f $(USRLIB_RELPATH)$(<F).$(LIB_VERSION) $(usrlibdir)/$(<F)
 
-# FIXME: plugins are installed to subdirs
-#        and for compatibility links in libdir are created
-#        when the code is fixed links could be removed.
+install_dm_lib: $(LIB_SHARED)
+	$(INSTALL_PROGRAM) -D $< $(libdir)/device-mapper/$(<F).$(LIB_VERSION)
+
 install_dm_plugin: $(LIB_SHARED)
 	$(INSTALL_PROGRAM) -D $< $(libdir)/device-mapper/$(<F)
-	$(LN_S) -f device-mapper/$(<F) $(libdir)/$(<F)
 
 install_lvm2_plugin: $(LIB_SHARED)
 	$(INSTALL_PROGRAM) -D $< $(libdir)/lvm2/$(<F)
-	$(LN_S) -f lvm2/$(<F) $(libdir)/$(<F)
-	$(LN_S) -f $(<F) $(libdir)/$(<F).$(LIB_VERSION)
 endif
 
 $(LIB_STATIC): $(OBJECTS)
Index: LVM2.2.02.120/daemons/dmeventd/plugins/mirror/Makefile.in
===================================================================
--- LVM2.2.02.120.orig/daemons/dmeventd/plugins/mirror/Makefile.in
+++ LVM2.2.02.120/daemons/dmeventd/plugins/mirror/Makefile.in
@@ -17,7 +17,7 @@ top_srcdir = @top_srcdir@
 top_builddir = @top_builddir@
 
 INCLUDES += -I$(top_srcdir)/daemons/dmeventd/plugins/lvm2
-CLDFLAGS += -L$(top_builddir)/daemons/dmeventd/plugins/lvm2
+CLDFLAGS += -L$(top_builddir)/daemons/dmeventd/plugins/lvm2 -Wl,-rpath,'$$ORIGIN'
 
 SOURCES = dmeventd_mirror.c
 
Index: LVM2.2.02.120/daemons/dmeventd/plugins/raid/Makefile.in
===================================================================
--- LVM2.2.02.120.orig/daemons/dmeventd/plugins/raid/Makefile.in
+++ LVM2.2.02.120/daemons/dmeventd/plugins/raid/Makefile.in
@@ -29,6 +29,7 @@ CFLOW_LIST_TARGET = $(LIB_NAME).cflow
 
 include $(top_builddir)/make.tmpl
 
+CLDFLAGS += -Wl,-rpath,'$$ORIGIN'
 LIBS += -ldevmapper-event-lvm2 -ldevmapper
 
 install_lvm2: install_dm_plugin
Index: LVM2.2.02.120/daemons/dmeventd/plugins/snapshot/Makefile.in
===================================================================
--- LVM2.2.02.120.orig/daemons/dmeventd/plugins/snapshot/Makefile.in
+++ LVM2.2.02.120/daemons/dmeventd/plugins/snapshot/Makefile.in
@@ -17,7 +17,7 @@ top_srcdir = @top_srcdir@
 top_builddir = @top_builddir@
 
 INCLUDES += -I$(top_srcdir)/daemons/dmeventd/plugins/lvm2
-CLDFLAGS += -L$(top_builddir)/daemons/dmeventd/plugins/lvm2
+CLDFLAGS += -L$(top_builddir)/daemons/dmeventd/plugins/lvm2 -Wl,-rpath,'$$ORIGIN'
 
 SOURCES = dmeventd_snapshot.c
 
Index: LVM2.2.02.120/daemons/dmeventd/plugins/thin/Makefile.in
===================================================================
--- LVM2.2.02.120.orig/daemons/dmeventd/plugins/thin/Makefile.in
+++ LVM2.2.02.120/daemons/dmeventd/plugins/thin/Makefile.in
@@ -29,6 +29,7 @@ CFLOW_LIST_TARGET = $(LIB_NAME).cflow
 
 include $(top_builddir)/make.tmpl
 
+CLDFLAGS += -Wl,-rpath,'$$ORIGIN'
 LIBS += -ldevmapper-event-lvm2 -ldevmapper
 
 install_lvm2: install_dm_plugin
Index: LVM2.2.02.120/daemons/dmeventd/plugins/lvm2/Makefile.in
===================================================================
--- LVM2.2.02.120.orig/daemons/dmeventd/plugins/lvm2/Makefile.in
+++ LVM2.2.02.120/daemons/dmeventd/plugins/lvm2/Makefile.in
@@ -26,6 +26,6 @@ include $(top_builddir)/make.tmpl
 
 LIBS += @LVM2CMD_LIB@ -ldevmapper $(PTHREAD_LIBS)
 
-install_lvm2: install_lib_shared
+install_lvm2: install_dm_lib
 
 install: install_lvm2
